asv_bench/benchmarks/io/style.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+
+from pandas import DataFrame
+
+
+class RenderApply:
+
+    params = [[12, 24, 36], [12, 120, 1200]]
+    param_names = ["cols", "rows"]
+
+    def setup(self, cols, rows):
+        self.df = DataFrame(
+            np.random.randn(rows, cols),
+            columns=[f"float_{i+1}" for i in range(cols)],
+            index=[f"row_{i+1}" for i in range(rows)],
+        )
+        self._style_apply()
+
+    def time_apply(self, cols, rows):
+        # skip known performance decrease: 36x1200
+        # reason unknown, it took less time in v1.0.4 compared to 36x120
+        #       rows ----------------------------------
+        #  cols       12          120           1200
+        #   36    12.4±0.3μs   12.0±0.1μs   11.8±0.09μs
+        if rows == 1200 and cols == 36:
```

Comment:
why?

Reply:
Well I though that would be nice but if that's not necessary, I can remove it.